### PR TITLE
Add default paths for WPT tests

### LINKED ISSUE #23133: Use pregenerated certs and keys for WPT
We should [set the following defaults](https://github.com/servo/servo/blob/681c0ebf4b5a4b5abf0843986036e3594cc193b8/tests/wpt/run.py#L51-L79) for running WPT tests:
* `--ca-cert-path=tests\wpt\web-platform-tests\tools\certs\cacert.pem`
* ` --host-key-path=tests\wpt\web-platform-tests\tools\certs\web-platform.test.key`
* `--host-cert-path=tests\wpt\web-platform-tests\tools\certs\web-platform.test.pem`
* `--ssl-type=pregenerated`

The best way to test that these changes are valid is to run `./mach test-wpt tests/wpt/mozilla/tests/mozilla/secure.https.html`. If it reports no unexpected results, then they are working.

### PR DESCRIPTION
**Pull request by soniasingla**
Set the following default configs for WPT tests.
```--ca-cert-path=tests\wpt\web-platform-tests\tools\certs\cacert.pem
--host-key-path=tests\wpt\web-platform-tests\tools\certs\web-platform.test.key
--host-cert-path=tests\wpt\web-platform-tests\tools\certs\web-platform.test.pem
--ssl-type=pregenerated
```

- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors
- [X] These changes fix #23133

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/23174)

### CODE REVIEW

File: tests/wpt/run.py [hunk 1]
```diff
@@ -74,6 +78,18 @@ def set_defaults(kwargs):
     if kwargs["processes"] is None:
         kwargs["processes"] = multiprocessing.cpu_count()
 
+    if kwargs["ca_cert_path"] is None:
```
Comment (CYBAI) [inline tests/wpt/run.py]: Maybe we'll want to check `"ca-cert-path" in paths` like the following one  👀? (Same for other new added args)

https://github.com/servo/servo/blob/37259d216d7256e930fa954c2357a349933c39c3/tests/wpt/run.py#L62

Comment (jdm) [reply tests/wpt/run.py]: I see that the existing code has those checks, but I don't see a good reason for them.

**Comment by jdm** [comment]
One tidy check failed:
```
./tests/wpt/run.py:27: E124 closing bracket does not match visual indentation
```

**Pull request merged**